forms-flow-admin/src/components/dashboard/dashboard.tsx

```diff
@@ -310,7 +310,7 @@ export const InsightDashboard = React.memo((props: any) => {
               keyField="resourceId"
               data={authDashBoardList}
               columns={columns}
-              pagination={pagination}
+              pagination={authDashBoardList.length > 0 ? pagination : undefined}
```

Comment:
instead of passing undefined could you pass `false` to pagination

Reply:
okay